services/parser-database/connector.py

```diff
@@ -73,8 +101,10 @@ def save_keywords_in_memory(keywords, article):
         if keyword not in keywords_in_memory:
             keywords_in_memory[keyword] = []
         keywords_in_memory[keyword].append({
+            "id": article["id"],
             "articleNumber": article["articleNumber"],
-            "frequency": frequency
+            "frequency": frequency,
+            "wordCount": article["wordCount"]
```

Comment:
Why are you adding an ID to the keywords and a word count? This information is already available elsewhere.

Reply:
Where is this info available? Since I don't know in depth your code this was what I managed to figure out

Reply:
The article dictionary